### PR TITLE
Fix outputs, update a deprecated method, and small formatting changes

### PR DESCRIPTION
**Pull request by bmitc**
The `Formatting-outputs.ipynb` sample for F# was not rendering correctly in GitHub.

-----

image

-----

So I fixed it, including some updates to an outdated method call and some small spelling and formatting changes.

I checked, and it looks like it is rendering properly now. I left the outputs in, as that seems to be what the previous notebook was doing.

-----

image

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/interactive?pullRequest=2211) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/interactive?pullRequest=2211) before we can accept your contribution.<br/><br/>:x: bmitc [sign now](https://cla.dotnetfoundation.org/dotnet/interactive?pullRequest=2211)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/interactive?pullRequest=2211) it.</sub>

File: samples/notebooks/fsharp/Docs/Formatting-outputs.ipynb [hunk 1]
```diff
@@ -58,18 +66,24 @@
...
+      ]
      },
-     "output_type": "unknown"
+     "metadata": {},
+     "output_type": "display_data"
     }
    ],
    "source": [
-    "Formatter.SetPreferredMimeTypeFor(typeof<obj>, \"text/plain\")\n",
+    "Formatter.SetPreferredMimeTypesFor(typeof<obj>, \"text/plain\")\n",
```
Comment (bmitc) [inline samples/notebooks/fsharp/Docs/Formatting-outputs.ipynb:86]: `Formatter.SetPreferredMimeTypeFor` doesn't seem to exist anymore (note singular on `Type`, so I updated this.

I contemplated doing `Formatter.SetPreferredMimeTypesFor(typeof<obj>, [|"text/plain"|])`.

File: samples/notebooks/fsharp/Docs/Formatting-outputs.ipynb [hunk 2]
```diff
@@ -130,7 +171,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can use styling to preserve the preformatting of such content."
+    "You can use styling to preserve the pre-formatting of such content."
```
Comment (jonsequitur) [inline samples/notebooks/fsharp/Docs/Formatting-outputs.ipynb:174]:
```suggestion
    "You can use styling to preserve the preformatting of such content."
```

Comment (bmitc) [reply samples/notebooks/fsharp/Docs/Formatting-outputs.ipynb]: Thanks! I think my spell checker had flagged that. Haha. Sorry about that. [Fixed here.](https://github.com/dotnet/interactive/pull/2211/commits/080f2de0583fb05eccfb83e3c405801a51e544ab)

**Review by jonsequitur — APPROVED** [review]

**Review by colombod — APPROVED** [review]

**Pull request merged**